backend/routes/users.js

```diff
@@ -50,11 +50,27 @@ users.get('/:id', async ctx => {
 });
 
 users.patch('/:id', parsers.json, async (ctx) => {
-    try{ctx.body = await Users.query()
-        .patch({isVerified: true})
-        .findById(ctx.params.id)}
+    let response = {}
+    try{
+        let targetUser = await Users.query()
+        .findById(ctx.params.id)
+
+        //currently, we have to have dateRegistered and lastLogin in patch request or it won't work
+        let patchedPayload = Object.assign(ctx.request.body, {dateRegistered: targetUser.dateRegistered, lastLogin: 0})
+
+        //actual patch request
+        await Users.query()
+        .patch(patchedPayload)
```

Comment:
So this means the ORM catches if we have any weird stuff in the request, right? That's cool. I'm trying to think if there is a bug where that doesn't help us, which we'd need to validate the user data for.. I think let's cross that bridge when we get there but it's probably worth to keep in mind